### PR TITLE
docs: translate notes.

### PR DESCRIPTION
**Pull request by appleboy**
cc @lunny @tboerger

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: content/doc/installation/from-source.zh-cn.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ git checkout v1.0.0
 
 我们支持两种方式进行编译，Make 工具 和 Go 工具。不过我们推荐使用 Make工具，因为他将会给出更多的编译选项。
 
+**Note**: We recommend the Go version 1.6 or higher because we are using vendoring and we don't set the required env variable for 1.5 anywhere.
```
Comment (Bwko) [inline content/doc/installation/from-source.zh-cn.md:56]: @lunny Could you translate this?

Comment (lunny) [reply content/doc/installation/from-source.zh-cn.md:56]: Yes, I will after this one is merged.

**Comment by Bwko** [comment]
Trusted LGTM

**Pull request merged**